x-pack/plugins/cases/server/common/types/case.ts

```diff
@@ -47,6 +50,15 @@ export interface CasePersistedAttributes {
 }
 
 export type CaseTransformedAttributes = CaseAttributes;
+export const CaseTransformedAttributesRt = CaseAttributesRt;
+
+export const getPartialCaseTransformedAttributesRt = (): Type<Partial<CaseAttributes>> => {
```

Comment:
This function takes all `CaseAttributesRt` props and makes them optional. The produced type will strip out unknown fields. @jonathan-buttner if you have a better idea let me know.

Reply:
I thought about your suggestion but I wanted to avoid running any code when importing the file and run it only when need it. What do you think of it? Is it ok to run code when importing types?

Reply:
Sounds good!

Reply:
Oh never mind I thought this was the top level `common`, there's no `type` calls here.